HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/Core/GiniHealthAPI.swift

```diff
@@ -39,7 +39,7 @@ public final class GiniHealthAPI {
      *
      */
     public func paymentService() -> PaymentService {
-        return payService ?? PaymentService(sessionManager: SessionManager(userDomain: .default), apiDomain: .default)
+        return payService ?? PaymentService(sessionManager: SessionManager(userDomain: .default), apiDomain: .default, apiVersion: 4)
```

Comment:
Suggestion: Maybe add this default apiVersion as a Constant somewhere globally so that we can modify easier in the future